appyx-interactions/common/src/commonMain/kotlin/com/bumble/appyx/interactions/core/DraggableChildren.kt

```diff
@@ -82,6 +84,11 @@ fun <InteractionTarget : Any, ModelState : Any> DraggableAppyxComponent(
                     clipToBounds = clipToBounds
                 )
             }
+            .onPointerEvent {
+                if (it.type == PointerEventType.Release && appyxComponent.isDragging()) {
+                    appyxComponent.onDragEnd()
```

Comment:
maybe we can always report to component with `onRelease()` and check internally if it's being dragged or not. This way we don't need to expose the state